llm_on_ray/inference/inference_config.py

```diff
@@ -112,6 +112,27 @@ class ModelDescription(BaseModel):
     input_processor: str = "AutoProcessor"
     model_loader: str = "AutoModel"
 
+    chat_model_with_image: bool = False
+    chat_template: Union[str, None] = None
+    default_chat_template: str = (
```

Comment:
Can you add a test for this default chat template to make sure the output is expected.